chart/files/pod-template-file.kubernetes-helm-yaml

```diff
@@ -80,6 +80,9 @@ spec:
     {{- if .Values.workers.extraContainers }}
       {{- toYaml .Values.workers.extraContainers | nindent 4 }}
     {{- end }}
+  {{- if .Values.workers.priorityClassName }}
+  priorityClassName: {{ .Values.workers.priorityClassName }}
+  {{- end }}
```

Comment:
I think we should also have a priority class definition in the code base. Otherwise users will be expected to create their own priorityclasses and then only run this part.

Reply:
Sure, thank you. Will create one soon